foundation.xml

```diff
@@ -10,7 +10,7 @@
   <spawns>
     <spawn team="blue-team" region="blue-spawn" yaw="-90" kit="default-kit"/>
     <spawn team="red-team" region="red-spawn" yaw="90" kit="default-kit"/>
-    <default region="default-spawn" yaw="180"/>
+    <spawn defult="true" region="default-spawn" yaw="180"/>
```

Comment:
it's supposed to be how the pr says, default spawn no longer exists, it's an attrubute now